security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -55,7 +55,7 @@ const (
 	// IngressSecretNamespace the namespace of kubernetes secrets to watch.
 	ingressSecretNamespace = "INGRESS_GATEWAY_NAMESPACE"
 
-	// IngressGatewaySdsCaSuffix is the suffix of the sds resource name for root CA. All resource
+	// SdsCaSuffix is the suffix of the sds resource name for root CA. All resource
 	// names for ingress gateway root certs end with "-cacert".
 	IngressGatewaySdsCaSuffix = "-cacert"
```

Comment:
How come lint is passing here?

Reply:
I meant to change it. Will do now

Reply:
Ah, this same as that in `pilot/pkg/security/model/authentication.go`, meant not to change this one